docs/src/content/docs/it/reference/configuration.mdx

```diff
@@ -459,33 +496,33 @@ interface HeadConfig {
 **tipo:** `boolean`  
 **predefinito:** `false`
 
-Controlla se il piè di pagina mostra quando è stata aggiornata l'ultima volta la pagina.
+Controlla se il footer mostra quando la pagina è stata aggiornata l'ultima volta.
 
-Per impostazione predefinita, questa funzionalità si basa sulla cronologia Git del repository e potrebbe non essere accurata su alcune piattaforme di distribuzione che eseguono [cloni superficiali](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Una pagina può sovrascrivere questa impostazione o la data basata su Git utilizzando il [campo frontmatter `lastUpdated`](/it/reference/frontmatter/#lastupdated).
+Per impostazione predefinita, questa funzionalità si basa sulla cronologia Git del tuo repository e potrebbe non essere accurata su alcune piattaforme di distribuzione che eseguono [cloni superficiali](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt). Una pagina può sovrascrivere questa impostazione o la data basata su Git utilizzando il [campo frontmatter `lastUpdated`](/reference/frontmatter/#lastupdated).
 
 ### `pagination`
 
 **tipo:** `boolean`  
 **predefinito:** `true`
 
-Definisci se il piè di pagina deve includere i collegamenti alla pagina precedente e successiva.
+Definisce se il footer deve includere i link alla pagina precedente e successiva.
 
-Una pagina può sovrascrivere questa impostazione o il testo del collegamento e/o l'URL utilizzando i campi frontmatter [`prev`](/it/reference/frontmatter/#prev) e [`next`](/it/reference/frontmatter/#next).
+Una pagina può sovrascrivere questa impostazione o il testo del link e/o l'URL utilizzando i campi frontmatter [`prev`](/reference/frontmatter/#prev) e [`next`](/reference/frontmatter/#next).
 
 ### `favicon`
 
 **tipo:** `string`  
 **predefinito:** `'/favicon.svg'`
 
-Imposta il percorso della favicon predefinita per il tuo sito web che dovrebbe trovarsi nella directory `public/` ed essere valido (`.ico`, `.gif`, `.jpg`, `.png` o `.svg`) file di icone.
+Imposta il percorso del favicon predefinito per il tuo sito web che dovrebbe essere posizionato nella directory `public/` ed essere un file di icona valido (`.ico`, `.gif`, `.jpg`, `.png`, o `.svg`).
 
 ```js
 starlight({
   favicon: '/images/favicon.svg',
 }),
 ```
 
-Se devi impostare varianti aggiuntive o favicon di fallback, puoi aggiungere tag utilizzando l'[opzione`head`](#head):
+Se hai bisogno di impostare varianti aggiuntive o favicon di fallback, puoi aggiungere tag utilizzando l'[opzione `head`](#head):
```

Comment:
Looks like the change from https://github.com/withastro/starlight/pull/1627 is missing in the code example below.

Reply:
will fix in the next commit